scripts/weeklyInteractionScript.ts

```diff
@@ -0,0 +1,29 @@
+import { appConfig } from '../appConfig';
+import { sendCheckInDM } from '../utils/buttons';
+import { generatePairs } from '../utils/generatePairs';
+import { startConversations } from '../utils/startConversations';
+import { StatusDataManager } from '../database/statusDataManager';
+
+const interactWithBuddies = async () => {
```

Comment:
this is the script the GitHub Action will run every week. this should be straightforward